packages/stash/src/react/isArrayEqual.ts

```diff
@@ -0,0 +1,8 @@
+export function isArrayEqual(a: readonly unknown[], b: readonly unknown[]): boolean {
```

Comment:
this is specifically optimized for comparing `Object.values(getRecords(...))` from stash, where we can expect stable references to objects, but not necessarily the outer records map